### ISSUE TITLE
Clicking on a checkbox throws 'Cannot read property 'dispatch' of undefined' error

### ISSUE THREAD

**Issue by iychiang** [Raigecki/BulletJournal#3]
![image](https://user-images.githubusercontent.com/37087196/89146233-fc251f00-d507-11ea-9902-9579e96810fc.png)

**Comment by Raigecki** [comment]
This issue might require a whole restructuring of components
Inside Bullets.js:

```
import {BulletsContext} from './DayView'
import {BulletsContext} from './WeekView'

 <input 
     type="text"
     value={state.text}
     onChange={e => {
     setState({...state, text: e.target.value})                       
        bulletsContext.dispatch({
           type:'save', id: props.bullet.id, bullet: state 
        })                 
     }}
 />
```

**Comment by Raigecki** [comment]
Fixed with commit 16ae6085623c70e679f09b5fb98d55096d2184bd
